### ISSUE TITLE
Script @php artisan package:discover --ansi handling the post-autoload-dump event returned with error code 255  Installation failed, reverting ./composer.json to its original content.

### ISSUE THREAD

**Issue by bertofern** [Laravel-Backpack/FileManager#1]
Hi!
I am testing backpack 4 in laravel 7.

**I got the following error when executing:**
`$ composer require backpack/filemanager`

**The answer:**
```
$ composer require backpack/filemanager
Using version ^1.0 for backpack/filemanager
./composer.json has been updated
Loading composer repositories with package information
Updating dependencies (including require-dev)
Package operations: 6 installs, 0 updates, 0 removals
  - Installing psr/cache (1.0.1): Loading from cache
  - Installing league/flysystem-cached-adapter (1.0.9): Loading from cache
  - Installing studio-42/elfinder (2.1.56): Loading from cache
  - Installing barryvdh/elfinder-flysystem-driver (v0.2.1): Loading from cache
  - Installing barryvdh/laravel-elfinder (v0.4.5): Loading from cache
  - Installing backpack/filemanager (1.0.0): Downloading (100%)
league/flysystem-cached-adapter suggests installing ext-phpredis (Pure C implemented extension for PHP)
studio-42/elfinder suggests installing kunalvarma05/dropbox-php-sdk (VolumeDriver `Dropbox`2 require `kunalvarma05/dropbox-php-sdk.)
studio-42/elfinder suggests installing google/apiclient (VolumeDriver GoogleDrive require `google/apiclient:^2.0.)
studio-42/elfinder suggests installing nao-pon/flysystem-google-drive (require in GoogleDrive network volume mounting with Flysystem.)
barryvdh/elfinder-flysystem-driver suggests installing league/glide (1.x to display images through Glide)
Writing lock file
Generating optimized autoload files
ocramius/package-versions: Generating version class...
ocramius/package-versions: ...done generating version class
> Illuminate\Foundation\ComposerScripts::postAutoloadDump
> @php artisan package:discover --ansi

   Symfony\Component\ErrorHandler\Error\FatalError

  Trait 'Backpack\CRUD\app\Console\Commands\Traits\PrettyCommandOutput' not found

  at C:\xampp\htdocs\dev\componentes\laravel\admin\backpack-laravel-7\vendor\backpack\filemanager\src\Console\Commands\Install.php:11
     7| use Symfony\Component\Process\Exception\ProcessFailedException;
     8| use Symfony\Component\Process\Process;
     9| use Backpack\CRUD\app\Console\Commands\Traits\PrettyCommandOutput;
    10|
  > 11| class Install extends Command
    12| {
    13|     use PrettyCommandOutput;
    14|
    15|     protected $progressBar;
PHP Fatal error:  Trait 'Backpack\CRUD\app\Console\Commands\Traits\PrettyCommandOutput' not found in C:\xampp\htdocs\dev\componentes\laravel\admin\backpack-laravel-7\vendor\backpack\filemanager\src\Console\Commands\Install.php on line 11


   Whoops\Exception\ErrorException

  Trait 'Backpack\CRUD\app\Console\Commands\Traits\PrettyCommandOutput' not found

  at C:\xampp\htdocs\dev\componentes\laravel\admin\backpack-laravel-7\vendor\backpack\filemanager\src\Console\Commands\Install.php:11
     7| use Symfony\Component\Process\Exception\ProcessFailedException;
     8| use Symfony\Component\Process\Process;
     9| use Backpack\CRUD\app\Console\Commands\Traits\PrettyCommandOutput;
    10|
  > 11| class Install extends Command
    12| {
    13|     use PrettyCommandOutput;
    14|
    15|     protected $progressBar;

  1   C:\xampp\htdocs\dev\componentes\laravel\admin\backpack-laravel-7\vendor\filp\whoops\src\Whoops\Run.php:408
      Whoops\Run::handleError("Trait 'Backpack\CRUD\app\Console\Commands\Traits\PrettyCommandOutput' not found", "C:\xampp\htdocs\dev\componentes\laravel\admin\backpack-laravel-7\vendor\backpack\filemanager\src\Console\Commands\Install.php")

  2   [internal]:0
      Whoops\Run::handleShutdown()
Script @php artisan package:discover --ansi handling the post-autoload-dump event returned with error code 255

Installation failed, reverting ./composer.json to its original content.
```

**Comment by francescods** [comment]
Hi there,

I'm facing the same problem while installing the File Manager add-on (Laravel 7, php 7.4)

`> composer require backpack/filemanager`

`Symfony\Component\ErrorHandler\Error\FatalError`
  `Trait 'Backpack\CRUD\app\Console\Commands\Traits\PrettyCommandOutput' not found
  at vendor/backpack/filemanager/src/Console/Commands/Install.php:8`

**Comment by francescods** [comment]
It's only a missing file. 
After manually creating the file [PrettyCommandOutput.php](https://github.com/Laravel-Backpack/CRUD/blob/4.1/src/app/Console/Commands/Traits/PrettyCommandOutput.php) from 4.1 it worked.

**Comment by tabacitu** [comment]
Thanks for reporting this guys! I think this may be because filemanager is only meant to be installed on Backpack 4.1 and you were trying to install it on Backpack 4.0?

@bertofern were you running Backpack 4.0 too?

I've just launched [1.1.3](https://github.com/Laravel-Backpack/FileManager/releases/tag/1.1.3) which should fix this - it drops support for Backpack 4.0. Let me know if it doesn't or if anybody else encounters this issue, and we'll reopen.

Cheers!
